src/panoptes/utils/images/__init__.py

```diff
@@ -219,7 +219,25 @@ def _make_pretty_from_cr2(fname, title=None, timeout=15, **kwargs):
     except Exception as e:
         raise error.InvalidCommand(f"Error executing {script_name}: {e.output!r}\nCommand: {cmd}")
 
-    return fname.replace('cr2', 'jpg')
+    return fname.replace("cr2", "jpg")
+
+
+def _overlay_grid_image(fname_fits, fname_jpg, timeout=15, output_filename=None, **kwargs):
+    script_name = shutil.which("plot-constellations.sh")
+    cmd = [script_name, fname_fits, fname_jpg]
+
+    logger.debug(cmd)
+
+    try:
+        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+        logger.debug(output)
+    except Exception as e:
+        raise error.InvalidCommand(f"Error executing {script_name}: {e.output!r}\nCommand: {cmd}"
+
+    if output_filename is None: 
+        output_filename = fname_fits.replace(".fits", " -output.png")
+
+    return output_filename
```

Comment:
I don't think this is right, but I haven't been able to test it yet. I also probably need to include a doc string for this.